contrib/cirrus/lib.sh

```diff
@@ -345,6 +345,21 @@ install_test_configs(){
 remove_packaged_podman_files(){
     echo "Removing packaged podman files to prevent conflicts with source build and testing."
     req_env_var OS_RELEASE_ID
+
+    # If any binaries are resident they could cause unexpected pollution
+    for unit in io.podman.service io.podman.socket
+    do
+        for state in enabled active
+        do
+            if ! systemctl is-$state $unit
+            then
+                echo "Warning: $unit found $state prior to packaged-file removal"
+                systemctl disable $unit
+                systemctl stop $unit
```

Comment:
Should you `set +e` here, or add `|| true`? As best I can tell, the calling context does `set -e`, and these commands have a high probability of exiting nonzero.

Reply:
ya, I was struggling with weather or not to do this.  Thanks for twisting my arm :smile: